src/service/config/base-configuration.ts

```diff
@@ -86,19 +96,27 @@ export abstract class BaseConfiguration {
 
   abstract loadToken(): void;
 
+  abstract getFlowType(): FlowType;
+
   /**
    * Validates any user input and returns the stored user input from InputService if there were no errors
    * @returns {InputValues}
    */
   loadParsedInput(): InputValues {
     const inputs: InputValues = Container.get(InputService).inputs;
-    // validate any input that needs to be in a certain way
+
+    // customCommandTreatment values must be of the form: REGEX||REPLACE_REGEX
     inputs.customCommandTreatment?.forEach((cct) => {
       if (cct.split("||").length !== 2) {
         logAndThrow("Invalid format for custom command treatment. Required format: Regex||ReplaceRegex");
       }
     });
 
+    // startProject must be of the form: OWNER/PROJECT
+    if (inputs.startProject && inputs.startProject.split("/").length !== 2) {
```

Comment:
```suggestion
    if (inputs.startProject?.split("/").length !== 2) {
```

Reply:
`inputs.startProject?.split("/").length !== 2` returns `true` when `startProject` is undefined. Since it returns `true` it will throw an error. However, for our use case if `startProject` is undefined we need to skip validating it instead of throwing an error